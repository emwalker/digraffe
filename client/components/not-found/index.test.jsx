import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './index'

describe('<NotFound />', () => {
  const wrapper = shallow(<NotFound />)

  it('renders', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
