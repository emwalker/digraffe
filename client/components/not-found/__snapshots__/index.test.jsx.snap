// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          encodeSpecialCharacters={true}
          title="404 Page Not Found"
        />,
        <h2
          className="notFound"
        >
          404 Page Not Found
        </h2>,
        <IndexLink
          className="link"
          to="/"
        >
          go home
        </IndexLink>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "encodeSpecialCharacters": true,
          "title": "404 Page Not Found",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404 Page Not Found",
          "className": "notFound",
        },
        "ref": null,
        "rendered": "404 Page Not Found",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "go home",
          "className": "link",
          "to": "/",
        },
        "ref": null,
        "rendered": "go home",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            encodeSpecialCharacters={true}
            title="404 Page Not Found"
          />,
          <h2
            className="notFound"
          >
            404 Page Not Found
          </h2>,
          <IndexLink
            className="link"
            to="/"
          >
            go home
          </IndexLink>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "encodeSpecialCharacters": true,
            "title": "404 Page Not Found",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404 Page Not Found",
            "className": "notFound",
          },
          "ref": null,
          "rendered": "404 Page Not Found",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "go home",
            "className": "link",
            "to": "/",
          },
          "ref": null,
          "rendered": "go home",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
